### PR TITLE
added routes

### PR DESCRIPTION
**Pull request by zordican**
Added routes

### CODE REVIEW

File: src/Components/StudentStyle.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```
Comment (Himangshu-101) [inline src/Components/StudentStyle.scss:5]: this block is redundant and avoid selecting the whole body or *

File: src/App.jsx [hunk 2]
```diff
@@ -2,21 +2,20 @@ import { BrowserRouter, Route, Routes } from "react-router-dom";
...
-          </Routes>
-        </div>
-      </BrowserRouter>
-    </>
+    <BrowserRouter>
+      <Navbar />
+      <div>
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route path="/login/student" element={<StudentLogin />}></Route>
```
Comment (Himangshu-101) [inline src/App.jsx:14]: use exact path here

File: src/App.jsx [hunk 3]
```diff
@@ -2,21 +2,20 @@ import { BrowserRouter, Route, Routes } from "react-router-dom";
 
 import Navbar from "./Components/Shared/Navbar/Navbar";
 import Home from "./Pages/Home/Home";
+import StudentLogin from "./Components/StudentLogin";
 
-function App() {
+const App = () => {
```
Comment (Himangshu-101) [inline src/App.jsx:7]: why did you change function to const . You can specify your reason here

File: src/Components/StudentStyle.scss
Comment (Himangshu-101) [inline src/Components/StudentStyle.scss:1]: rename it to module.scss file name

**Review by Himangshu-101 — CHANGES_REQUESTED** [review]

File: src/Components/StudentStyle.scss (hunk 1 above)
Comment (zordican) [reply src/Components/StudentStyle.scss:5]: okay bhaiya

File: src/App.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (zordican) [reply src/App.jsx:14]: okay bhaiya

File: src/Components/module.scss
Comment (hereisadi) [inline src/Components/module.scss:1]: found use of px. use relative font units instead.

File: src/App.jsx [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
-        <div>
-          <Routes>
-            <Route exact path="/" element={<Home />} />
-          </Routes>
-        </div>
-      </BrowserRouter>
-    </>
+    <BrowserRouter>
+      <Navbar />
+      <div>
```
Comment (hereisadi) [inline src/App.jsx:10]: no need to wrap routes inside div.

File: src/Components/module.scss [hunk 5]
```diff
@@ -0,0 +1,87 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;700&display=swap");
```
Comment (hereisadi) [inline src/Components/module.scss:1]: redundant import. Poppins has already been imported in index.scss

File: src/Components/module.scss [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 16px;
+}
+
+.form label {
+  position: absolute;
+  font-family: "poppins", sans-serif;
```
Comment (hereisadi) [inline src/Components/module.scss:26]: use the variable ` --vyatha-font-1` for Poppins as specified in index.scss

**Review by hereisadi — CHANGES_REQUESTED** [review]

File: src/Components/StudentSignUp.jsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import "./StudentSignUp.module.scss";
+
+const StudentSignUp = () => {
+  return (
+    <>
+      <h1>Sign Up</h1>
+      <form action="">
+        <div className="form">
```
Comment (hereisadi) [inline src/Components/StudentSignUp.jsx:8]: you have to className like:
`className = {styles.form}`

File: src/Components/StudentSignUp.jsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
+import "./StudentSignUp.module.scss";
```
Comment (hereisadi) [inline src/Components/StudentSignUp.jsx:1]: correct way:
`import styles from "./StudentSignUp.module.scss";`

**Review by hereisadi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**